docs/contributing/architecture/app-layout.rst

```diff
@@ -1,9 +1,6 @@
 Pulp Platform Application Layout
 ================================
 
-This documentation is for Pulp Platform developers. For Plugin development,
-see the :doc:`../../plugins/plugin-writer/index`.
-
```

Comment:
I added a checklist item to https://pulp.plan.io/issues/4193 to re-add links to the plugin api docs.